### DISCUSSION TITLE
AZURE SIGN UP ISSUE USING GITHUB DEVELOPER PACK

### DISCUSSION THREAD

**Discussion by HarshilChangrani** [community/community#57276] [GitHub Education]
### Hi everyone,

I was trying to signup on AZURE using GIthub Students developer pack but I was not able to sign up I was getting an error saying We could not confirm your status as a student. Click [here](https://go.microsoft.com/fwLink/p/?LinkID=2136428&clcid=0x409) to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students. 

<img width="1440" alt="Screenshot 2023-06-06 at 6 16 20 AM" src="https://github.com/community/community/assets/93158369/7ee22092-8c0b-4e2b-8604-29d32548b598">

I already have the github Education pack and the benefits also Its a new account not a renew

<img width="1440" alt="Screenshot 2023-06-06 at 6 18 55 AM" src="https://github.com/community/community/assets/93158369/5451fd4b-93c4-4495-9bbf-cdbe78bc2d35">

I also tried to verify using my student email id but there was an error again saying - Your email domain is not currently registered with us. You can choose another verification method.

<img width="1440" alt="Screenshot 2023-06-06 at 6 37 56 AM" src="https://github.com/community/community/assets/93158369/235250c9-2884-474c-ad4a-a6061b288898">

I also asked Azure for the 25-Digit Verification code but I got 0 response

<img width="1440" alt="Screenshot 2023-06-06 at 6 21 55 AM" src="https://github.com/community/community/assets/93158369/b01feaa5-daef-42b8-9519-48c3decbc4e3">

I tried to contact AZURE support and Github Support but I got 0 response from there side.

<img width="1440" alt="Screenshot 2023-06-06 at 6 24 16 AM" src="https://github.com/community/community/assets/93158369/9b20d167-3095-402e-8394-17411d265ea2">

<img width="1440" alt="Screenshot 2023-06-06 at 6 24 44 AM" src="https://github.com/community/community/assets/93158369/10e84fdb-c39a-4b2a-8d3d-e3c718ac0a00">

Please Help me resolve this issue I would be really thank full My Github ID - @ITzHARSHIL

**Comment by SOULREDOUANE** [comment]
did you solve your problem. 
i have the same issues like you?

**Reply by Mayankraj2008** [reply]
Yes it was resolved. Just tweet microsoft azure support and they will do everything

**Reply by y0geshx** [reply]
> Yes it was resolved. Just tweet microsoft azure support and they will do everything

You asked in Twitter to resolve issues 🫡
Did they replied..

**Reply by ImSound1** [reply]
Same issue here

**Reply by ArvashV** [reply]
Same here man, any solutions?

**Reply by ImSound1** [reply]
Nope, I did not found anything

**Reply by Mayankraj2008** [reply]
Yes, they did resolve it by sending me the 25(24,25 what every) digit code they just asked for a screen recording and gave me the code. But this took 1-2 weeks

**Reply by ArvashV** [reply]
can I know where to contact them at?

**Reply by Mayankraj2008** [reply]
Just tweet them at there twitter handle @AzureSupport

**Comment by HAMZA-420** [comment]
It looks like you're encountering an issue while trying to sign up for Azure using the GitHub Student Developer Pack. The error you're seeing typically occurs when Azure is unable to automatically verify your student status through GitHub. Here’s what you can do to resolve this issue:

### 1. **Begin the Academic Verification Process**
   - **Click the Provided Link**: The message you received should include a link to start the academic verification process. Click on that link to begin.
   - **Submit Documentation**: You may be asked to submit documentation that proves your student status, such as a student ID, a transcript, or an enrollment letter. Make sure the documentation is clear and includes your name, school, and current enrollment status.
   - **Wait for Verification**: The verification process may take a few days, as mentioned in the error message. During this time, Microsoft will review the documents you submitted.

### 2. **Ensure Your GitHub Account is Properly Linked**
   - **GitHub Account**: Ensure that your GitHub account is properly linked with your student status. Visit the [GitHub Education page](https://education.github.com/pack) to confirm that you have access to the GitHub Student Developer Pack.
   - **Check Email and School Domain**: Make sure that the email address associated with your GitHub account is the one linked to your academic institution. Sometimes using a non-school email can cause verification issues.

### 3. **Contact Microsoft Support**
   - **Microsoft Support**: If the verification process takes longer than expected, or if you encounter any issues, contact Microsoft Support for assistance. You can do this by visiting the [Microsoft Azure support page](https://azure.microsoft.com/en-us/support/) and explaining your situation.
   - **Provide Necessary Details**: When contacting support, be ready to provide any additional documentation or details that can help expedite the verification process.

### 4. **Retry After Verification**
   - **Sign Up Again**: Once your student status is verified, try signing up for Azure again using the GitHub Student Developer Pack. The process should go smoothly if the verification is successful.

### 5. **Check GitHub and Azure Status**
   - **Platform Issues**: Sometimes, issues can be related to temporary glitches on GitHub or Azure’s side. Check their status pages ([GitHub Status](https://www.githubstatus.com/), [Azure Status](https://status.azure.com/en-us/status)) to ensure there are no ongoing problems.

### 6. **Alternative Student Verification**
   - **University Email**: If the automatic verification is still not working, try signing up for Azure for Students directly using your university email address, which might offer a manual verification process.

By following these steps, you should be able to resolve the issue and successfully sign up for Azure using your GitHub Student Developer Pack.

**Comment by Vikash-Kumar-23** [comment]
i also have the same issue how to resolve it
